python/core/auto_generated/qgsdistancearea.sip.in

```diff
@@ -189,23 +189,25 @@ Measures the perimeter of a polygon geometry.
 .. versionadded:: 2.12
 %End
 
-    double measureLine( const QVector<QgsPointXY> &points ) const;
+    double measureLine( const QVector<QgsPointXY> &points, bool forceCartesian = false ) const;
```

Comment:
I think we can avoid these new "forceCartesian" arguments here -- what we could do instead is from qgsmeasuredialog.cpp call `mDa.setEllipsoid( QString() )`. That will disable ellipsoidal calculation. And then to re-enable the ellipsoidal calculation you'd call `mDa.setEllipsoid( QgsProject::instance()->ellipsoid() )`. 

I'd rather avoid the extra argument here and keep the QgsDistanceArea simpler (I'm sure you'll agree it's already quite complex!)

Reply:
Done (although I had to use `mDa.setEllipsoid( GEO_NONE )` for this to work)